types/nock/nock-tests.ts

```diff
@@ -604,6 +604,10 @@ if (!scope.isDone()) {
 }
 console.error('pending mocks: %j', nock.pendingMocks());
 
+/// .activeMocks()
+strings[] = nock('http://example.com').activeMocks();
+strings[] = nock.activeMocks();
```

Comment:
@tagyoureit the tests are failing because this isn't valid TS.
To run the tests locally, assuming you've already run `npm install`, you run `npm run lint nock`. [(Docs)](https://github.com/DefinitelyTyped/DefinitelyTyped#lint)

These lines needs to be updated to the following:
```ts
nock.activeMocks(); // $ExpectType string[]
nock('http://example.com').activeMocks(); // $ExpectType string[]
```